src/geometryXYVxVy/time_integration/predcorr.cpp

```diff
@@ -64,9 +65,9 @@ DFieldSpVxVyXY PredCorr::operator()(
                 .with("time_saved", iter_time)
                 .with("fdistribu", allfdistribu_host)
                 .with("electrostatic_potential", electrostatic_potential_host);
-
         // copy fdistribu
         ddc::parallel_deepcopy(allfdistribu_half_t, allfdistribu_v2D_split);
+        Kokkos::Profiling::popRegion();
```

Comment:
The deepcopy here is not related to the `PDIWrite`. Shouldn't this be:
```suggestion
        Kokkos::Profiling::popRegion();
        // copy fdistribu
        ddc::parallel_deepcopy(allfdistribu_half_t, allfdistribu_v2D_split);
```